.env

```diff
@@ -0,0 +1,5 @@
+gmailUser = 'munibadonutshop@gmail.com'
```

Comment:
You have commited the ENV file, this is no no, You will need delete commits and recommit without this being uploading